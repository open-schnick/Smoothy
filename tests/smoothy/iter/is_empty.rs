use crate::failing_assertion;
use smoothy::{assert_that, IteratorAssertion};

#[test]
fn succeeds() {
    let vec: Vec<String> = vec![];
    assert_that(vec).is_empty();
}

#[test]
fn fails() {
    failing_assertion!({
        let vec: Vec<String> = vec!["Hello".to_string()];
        assert_that(vec).is_empty();
    });
}
