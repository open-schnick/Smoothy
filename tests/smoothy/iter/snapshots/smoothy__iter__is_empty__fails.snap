---
source: tests/smoothy/iter/is_empty.rs
expression: assertion_failed_output
---
[31mAssertion failed![0m

Expected
  [31m["Hello"][0m
to be empty
